app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      include Response
+      include ExceptionHandler
+
+      def show
+        json_response(find_volunteer)
+      end
+
+      def create
+        json_response(Volunteer.create!(volunteer_params))
+      end
+
+      def update
+        json_response(find_volunteer.update!(volunteer_params))
+      end
+
+      def destroy
+        find_volunteer.destroy!
+        head :no_content
+      end
+
+      def get
+        volunteers_json = fetchVolunteerJson;
+
+        respond_to do |format|
+          format.json do
+            render(json: volunteers_json)
+          end
+        end
+      end
+
+      private
+
+      def find_volunteer
+        Volunteer.find(params[:id])
```

Comment:
This should read @volunteer = Volunteer.find!(params[:id]) so the variable can be reused in update and delete functions